apps/graphql/src/Schema.js

```diff
@@ -7,17 +7,17 @@ import SearchReturnItineraries from './apps/itinerary/queries/SearchReturnItiner
 import LocationsByTerm from './apps/location/queries/LocationsByTerm';
 import CustomerBookings from './apps/booking/queries/CustomerBookings';
 import BookingDetail from './apps/booking/queries/BookingDetail';
-import SaveBooking from './apps/booking/mutations/SaveBooking';
-import ConfirmPayment from './apps/booking/mutations/ConfirmPayment';
+import SaveBooking from './apps/mmb/mutations/SaveBooking';
```

Comment:
can we use `manageMyBooking` instead of `mmb`. This abbreviation might be confusing for external people reading this code.

Reply:
Done!